src/test/java/io/confluent/idesidecar/restapi/resources/KafkaConsumeResourceIT.java

```diff
@@ -83,15 +105,14 @@ void testConfluentLocalContainer() {
           .body().asString();
 
       assertConsumerRecords(rows, sampleRecords);
-    }
+      deleteLocalConnection(connectionId);
```

Comment:
This connection will not be removed if the tests fail. There are two options:
1. Add an `@AfterEach` method that removes all connections; or
2. Add an `@AfterEach` method that removes all connections that were created in `setupTestEnvironment(...) -- probably would require bookkeeping the created connection IDs in a field`.

The first would be the easiest with least bookkeeping, but if that's not feasible for some reason then the second should be doable.

Reply:
Done.